SingularityService/src/main/java/com/hubspot/singularity/data/history/HistoryJDBI.java

```diff
@@ -29,6 +29,9 @@
   @SqlQuery("SELECT bytes FROM taskHistory WHERE taskId = :taskId")
   byte[] getTaskHistoryForTask(@Bind("taskId") String taskId);
 
+  @SqlQuery("SELECT bytes FROM taskHistory WHERE requestId = :requestId AND runId = :runId")
+  byte[] getTaskHistoryForTaskByRunId(@Bind("requestId") String requestId, @Bind("runId") String runId);
```

Comment:
The one on the line for this comment thread:
`SELECT bytes FROM taskHistory WHERE requestId = :requestId AND runId = :runId`